File: pclassic/2017s/stubs/ImageLock.py
# Fill out the body of this method, img is a 2d array (list of lists)
# where the pixel value at (x, y) is given by img(x, y)
def isCat(img):
    return True

# Make sure this method hasn't changed before submitting!
if __name__ == "__main__":
    with open("ImageLockIN.txt", "r") as f:
        while True:
            s = f.readline()
            if s == "":
                break
            s = s.split(" ")
            W = int(s[0])
            H = int(s[1])
            img = [ [ 0 for i in range(H) ] for j in range(W) ]
            for y in range(H):
                s = f.readline()
                s = s.split(" ")
                for x in range(W):
                    img[x][y] = int(s[x])

            if isCat(img):
                print('cat')
            else:
                print('random')

File: pclassic/2017s/stubs/ImageLock.java
import java.util.*;
import java.io.*;
public class ImageLock {
    public static void main(String[] args) throws FileNotFoundException {
        // Before submitting, make sure the main method hasn't been changed!
        Scanner sc = new Scanner(new FileReader("ImageLockIN.txt"));
        while(sc.hasNext()) {
            int width = sc.nextInt();
            int height = sc.nextInt();
            int[][] image = new int[width][height];
            for (int i = 0; i < height; i++) {
              for (int j = 0; j < width; j++) {
                image[j][i] = sc.nextInt();
              }
            }
            if (isCat(image)) {
              System.out.println("cat");
            } else {
              System.out.println("random");
            }
        }
        sc.close();
    }

  //Fill out the body of this method
  public static boolean isCat(int[][] image) {
    boolean ans = true;
    return ans;
  }
}
